core-tests/shared/src/test/scala/zio/ZIOSpec.scala

```diff
@@ -1954,16 +1954,16 @@ object ZIOSpec extends ZIOBaseSpec {
         } yield assert(res)(equalTo(42))
       } @@ zioTag(interruption),
       testM("timeout a long computation") {
-        val io = (clock.sleep(5.seconds) *> IO.succeed(true)).timeout(10.millis)
+        val io = (clock.sleep(fromSeconds(5)) *> IO.succeed(true)).timeout(fromMillis(10))
         assertM(Live.live(io))(isNone)
       },
       testM("timeout repetition of uninterruptible effect") {
         val effect = ZIO.unit.uninterruptible.forever
 
-        assertM(Live.live(effect.timeout(1.second)))(isNone)
+        assertM(Live.live(effect.timeout(fromSeconds(1))))(isNone)
       } @@ jvmOnly @@ zioTag(interruption),
       testM("timeout in uninterruptible region") {
-        val effect = (ZIO.unit.timeout(Duration.Infinity)).uninterruptible
+        val effect = (ZIO.unit.timeout(zio.duration.infiniteDuration)).uninterruptible
```

Comment:
We can still keep `Duration.Infinity` / etc. There can be a type alias `Duration` inside `zio.duration` which delegates to `java.time.Duration`, then there can be a companion object with vals inside it and helpers. So overall almost no code should change in user-land.